infrastructure/local/peer-docker-compose.yml

```diff
@@ -34,8 +32,8 @@ services:
       ILP_ADDRESS: test.peer
       STREAM_SECRET: BjPXtnd00G2mRQwP/8ZpwyZASOch5sUXT5o0iR5b5wU=
       ADMIN_KEY: admin
-      PUBLIC_HOST: rafiki.peer
-      OPEN_PAYMENTS_URL: op.rafiki.peer
+      ADMIN_URL: http://rafiki.peer:3001
+      OPEN_PAYMENTS_URL: http://op.rafiki.peer:3003
```

Comment:
Lets change this to `ADMIN_HOST` and drop the `http://`

so it would be
```
ADMIN_HOST: rafiki.peer:3001
 ```

Reply:
Lets change this likewise for `OPEN_PAYMENTS_HOST` and I see the connector is also still on just port. Lets change that as well